### PR TITLE
[zh-cn]: create doc for "Fenced Frame API"

### PR DESCRIPTION
**Pull request by fuchunhui**

### CODE REVIEW

File: files/zh-cn/web/api/fenced_frame_api/index.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+  sourceCommit: f216422c99b6c7014e398803b70600501bce8a48
+---
+
+{{SeeCompatTable}}{{DefaultAPISidebar("Fenced Frame API")}}
+
+**Fenced Frame API** 为控制嵌入在 {{htmlelement("fencedframe")}} 元素中的内容提供功能。
+
+## 概念和用法
+
+网络上[隐私](/zh-CN/docs/Web/Privacy)和[安全](/zh-CN/docs/Web/Security)问题的一个主要来源是嵌入在 {{htmlelement("iframe")}} 元素中的内容。历史上，`<iframe>`被用于设置第三方 cookie，这些 cookie 可用于跨站点共享信息和追踪用户。此外，嵌入在 `<iframe>` 中的内容可以与其嵌入文档进行通信（例如，使用 {{domxref("Window.postMessage()")}}）。
```
Comment (T34-active) [inline files/zh-cn/web/api/fenced_frame_api/index.md:14]:
```suggestion
网络上[隐私](/zh-CN/docs/Web/Privacy)和[安全](/zh-CN/docs/Web/Security)问题的一个主要来源是嵌入在 {{htmlelement("iframe")}} 元素中的内容。历史上，`<iframe>` 被用于设置第三方 cookie，这些 cookie 可用于跨站点共享信息和追踪用户。此外，嵌入在 `<iframe>` 中的内容可以与其嵌入文档进行通信（例如，使用 {{domxref("Window.postMessage()")}}）。
```

File: files/zh-cn/web/api/fenced_frame_api/index.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+  - `allow-top-navigation-by-user-activation`
+
+### `beforeunload` 和 `unload` 事件
+
+[`beforeunload`](/zh-CN/docs/Web/API/Window/beforeunload_event) 和 [`unload`](/zh-CN/docs/Web/API/Window/unload_event) 事件不能在围栏框架上触发，因为它们可以以页面删除时间戳的形式泄露信息。实施旨在消除尽可能多的潜在泄漏。
+
+## 接口
+
+- {{domxref("FencedFrameConfig")}}
+  - : 表示 {{htmlelement("fencedframe")}} 的导航配置，即该框架中将显示什么内容。`FencedFrameConfig` 由诸如[受保护的受众 API](https://developers.google.com/privacy-sandbox/private-advertising/protected-audience)等来源返回，并设置为 {{domxref("HTMLFencedFrameElement.config")}} 的值。
```
Comment (T34-active) [inline files/zh-cn/web/api/fenced_frame_api/index.md:109]:
```suggestion
  - : 表示 {{htmlelement("fencedframe")}} 的导航配置，即该框架中将显示什么内容。`FencedFrameConfig` 由诸如[受保护的受众 API](https://developers.google.com/privacy-sandbox/private-advertising/protected-audience) 等来源返回，并设置为 {{domxref("HTMLFencedFrameElement.config")}} 的值。
```

File: files/zh-cn/web/api/fenced_frame_api/index.md [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+
+服务器必须为任何打算加载到 `<fencedframe>` 中或嵌入在 `<fencedframe>` 内的 `<iframe>` 中的文档设置一个值为 `fenced-frame` 的 {{httpheader("Supports-Loading-Mode")}} 响应头。
+
+```http
+Supports-Loading-Mode: fenced-frame
+```
+
+围栏框架对 HTTP 标头字段的其他影响如下：
+
+- [用户代理客户端提示](/zh-CN/docs/Web/HTTP/Client_hints#user-agent_client_hints)在封闭框架中不可用，因为它们依赖于[权限策略](/zh-CN/docs/Web/HTTP/Permissions_Policy)委托，这可能会被用来泄露数据。
```
Comment (T34-active) [inline files/zh-cn/web/api/fenced_frame_api/index.md:91]:
```suggestion
- [用户代理客户端提示](/zh-CN/docs/Web/HTTP/Client_hints#用户代理客户端提示)在封闭框架中不可用，因为它们依赖于[权限策略](/zh-CN/docs/Web/HTTP/Permissions_Policy)委托，这可能会被用来泄露数据。
```

File: files/zh-cn/web/api/fenced_frame_api/index.md [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+
+> **Note:** `FencedFrameConfig` 具有一个 {{domxref("FencedFrameConfig.setSharedStorageContext", "setSharedStorageContext()")}} 方法，用于将数据从嵌入文档传递到 `<fencedframe>` 共享存储中。例如，它可以在 {{domxref("Worklet")}} 中通过 `<fencedframe>` 访问，并用于生成报告。参见[共享存数 API](https://developers.google.com/privacy-sandbox/private-advertising/shared-storage)查看更多信息。
+
+### 在 `Fence` 对象上访问围栏框架功能
+
+在嵌入到 `<fencedframe>` 中的文档内，JavaScript 可以访问一个 {{domref("Window.fence")}} 属性，此属性为该文档返回一个 {{domref("Fence")}} 实例。此对象包含几个与围栏框架 API 功能特别相关的函数。例如，{{domref("Fence.reportEvent()")}} 提供了一种通过 [beacon](/zh-CN/docs/Web/API/Beacon_API) 向一个或多个指定的 URL 触发报告数据提交的方式，以便报告广告展示和点击情况。
+
+### 权限策略
+
+只有通过设置在 `<fencedframe>` 上的权限策略，才能启用专门设计为在其中使用的特定功能；在此上下文中，其他受策略控制的功能不可用。参见[围栏框架可用的权限策略](/zh-CN/docs/Web/HTML/Element/fencedframe#permissions_policies_available_to_fenced_frames)查看更多信息。
```
Comment (T34-active) [inline files/zh-cn/web/api/fenced_frame_api/index.md:73]:
```suggestion
只有通过设置在 `<fencedframe>` 上的权限策略，才能启用专门设计为在其中使用的特定功能；在此上下文中，其他受策略控制的功能不可用。参见[围栏框架可用的权限策略](/zh-CN/docs/Web/HTML/Element/fencedframe#围栏框架可用的权限策略)查看更多信息。
```

**Review by T34-active — COMMENTED** [review]
nits

File: files/zh-cn/web/api/fenced_frame_api/index.md [hunk 5]
```diff
@@ -0,0 +1,149 @@
+---
+title: Fenced Frame API
+slug: Web/API/Fenced_frame_API
+l10n:
+  sourceCommit: f216422c99b6c7014e398803b70600501bce8a48
+---
+
+{{SeeCompatTable}}{{DefaultAPISidebar("Fenced Frame API")}}
+
+**Fenced Frame API** 为控制嵌入在 {{htmlelement("fencedframe")}} 元素中的内容提供功能。
```
Comment (yin1999) [inline files/zh-cn/web/api/fenced_frame_api/index.md:10]:
```suggestion
**围栏框架 API**（Fenced Frame API）提供了用于控制嵌入在 {{htmlelement("fencedframe")}} 元素中的内容的功能。
```

File: files/zh-cn/web/api/fenced_frame_api/index.md [hunk 6]
```diff
@@ -0,0 +1,149 @@
...
+
+## 概念和用法
+
+网络上[隐私](/zh-CN/docs/Web/Privacy)和[安全](/zh-CN/docs/Web/Security)问题的一个主要来源是嵌入在 {{htmlelement("iframe")}} 元素中的内容。历史上，`<iframe>` 被用于设置第三方 cookie，这些 cookie 可用于跨站点共享信息和追踪用户。此外，嵌入在 `<iframe>` 中的内容可以与其嵌入文档进行通信（例如，使用 {{domxref("Window.postMessage()")}}）。
```
Comment (yin1999) [inline files/zh-cn/web/api/fenced_frame_api/index.md:14]:
```suggestion
Web 上[隐私](/zh-CN/docs/Web/Privacy)和[安全](/zh-CN/docs/Web/Security)问题的一个主要来源是嵌入在 {{htmlelement("iframe")}} 元素中的内容。历史上，`<iframe>` 被用于设置第三方 cookie，这些 cookie 可用于跨站点共享信息和追踪用户。此外，嵌入在 `<iframe>` 中的内容可以与其嵌入文档进行通信（例如，使用 {{domxref("Window.postMessage()")}}）。
```

File: files/zh-cn/web/api/fenced_frame_api/index.md [hunk 7]
```diff
@@ -0,0 +1,149 @@
...
+
+嵌入文档还可以使用脚本从 `<iframe>` 中读取各种形式的信息，例如，通过读取 `src` 属性中的嵌入 URL，你可能会获取到重要的跟踪/指纹数据，特别是当 URL 包含 [URL 参数](/zh-CN/docs/Web/URI#query)时。`<iframe>` 还可以访问嵌入上下文的 DOM，反之亦然。
+
+大多数现代浏览器都在研究存储分区机制，以便 cookie 数据不再用于跟踪（例如，参见[具有独立分区状态的 Cookie（CHIPS）](/zh-CN/docs/Web/Privacy/Privacy_sandbox/Partitioned_cookies)或 [Firefox 状态分区机制](/zh-CN/docs/Web/Privacy/State_Partitioning)）。
```
Comment (yin1999) [inline files/zh-cn/web/api/fenced_frame_api/index.md:18]:
```suggestion
大多数现代浏览器都在研究分区存储机制，以便 cookie 数据不再用于跟踪（有关示例请参见[具有独立分区状态的 Cookie（CHIPS）](/zh-CN/docs/Web/Privacy/Privacy_sandbox/Partitioned_cookies)或 [Firefox 状态分区](/zh-CN/docs/Web/Privacy/State_Partitioning)）。
```

File: files/zh-cn/web/api/fenced_frame_api/index.md [hunk 8]
```diff
@@ -0,0 +1,149 @@
...
+const frameConfig = await navigator.runAdAuction({
+  // 拍卖配置
+  resolveToConfig: true,
+});
+
+const frame = document.createElement("fencedframe");
+frame.config = frameConfig;
+```
+
+在调用 `runAdAuction()` 时，必须传入 `resolveToConfig: true` 以获得 `FencedFrameConfig` 对象。如果 `resolveToConfig` 设置为 `false`，则所得的 {{jsxref("Promise")}} 将兑现为一个不透明的 [URN](/zh-CN/docs/Web/URI#urns)（例如：`urn:uuid:c36973b5-e5d9-de59-e4c4-364f137b3c7a`），该 URN 只能在 `<iframe>` 中使用。
```
Comment (yin1999) [inline files/zh-cn/web/api/fenced_frame_api/index.md:58]:
```suggestion
在调用 `runAdAuction()` 时，必须传入 `resolveToConfig: true` 以获得 `FencedFrameConfig` 对象。如果 `resolveToConfig` 设置为 `false`，则所得的 {{jsxref("Promise")}} 将兑现为一个不透明 [URN](/zh-CN/docs/Web/URI#urns)（例如：`urn:uuid:c36973b5-e5d9-de59-e4c4-364f137b3c7a`），该 URN 只能在 `<iframe>` 中使用。
```

File: files/zh-cn/web/api/fenced_frame_api/index.md [hunk 9]
```diff
@@ -0,0 +1,149 @@
...
+
+无论哪种方式，浏览器都会存储一个包含要嵌入内容的目标位置的 URL，该 URL 与不透明的 URN 或 `FencedFrameConfig` 的内部 `url` 属性相对应。在嵌入上下文中运行的 JavaScript 无法读取 UR 值。
+
+> [!NOTE]
+> 在 `<iframe>` 中支持不透明的 URN，以便将现有实现轻松迁移到[隐私沙盒](https://developers.google.com/privacy-sandbox)。这种支持是暂时的，随着采用率的提高，未来将会移除。
```
Comment (yin1999) [inline files/zh-cn/web/api/fenced_frame_api/index.md:63]:
```suggestion
无论哪种方式，浏览器都会存储一个包含要嵌入内容的目标位置的 URL，该 URL 与不透明 URN 或 `FencedFrameConfig` 的内部 `url` 属性相对应。在嵌入上下文中运行的 JavaScript 无法读取 UR 值。

> [!NOTE]
> 在 `<iframe>` 中支持不透明 URN，以便将现有实现轻松迁移到[隐私沙盒](https://developers.google.com/privacy-sandbox)。这种支持是暂时的，随着采用率的提高，未来将会移除。
```

File: files/zh-cn/web/api/fenced_frame_api/index.md [hunk 10]
```diff
@@ -0,0 +1,149 @@
...
+
+> **Note:** `FencedFrameConfig` 具有一个 {{domxref("FencedFrameConfig.setSharedStorageContext", "setSharedStorageContext()")}} 方法，用于将数据从嵌入文档传递到 `<fencedframe>` 共享存储中。例如，它可以在 {{domxref("Worklet")}} 中通过 `<fencedframe>` 访问，并用于生成报告。参见[共享存数 API](https://developers.google.com/privacy-sandbox/private-advertising/shared-storage)查看更多信息。
+
+### 在 `Fence` 对象上访问围栏框架功能
+
+在嵌入到 `<fencedframe>` 中的文档内，JavaScript 可以访问一个 {{domref("Window.fence")}} 属性，此属性为该文档返回一个 {{domref("Fence")}} 实例。此对象包含几个与围栏框架 API 功能特别相关的函数。例如，{{domref("Fence.reportEvent()")}} 提供了一种通过 [beacon](/zh-CN/docs/Web/API/Beacon_API) 向一个或多个指定的 URL 触发报告数据提交的方式，以便报告广告展示和点击情况。
```
Comment (yin1999) [inline files/zh-cn/web/api/fenced_frame_api/index.md:69]:
```suggestion
在嵌入到 `<fencedframe>` 中的文档内，JavaScript 可以访问 {{domxref("Window.fence")}} 属性，此属性为该文档返回一个 {{domxref("Fence")}} 实例。此对象包含几个与围栏框架 API 功能特别相关的函数。例如，{{domxref("Fence.reportEvent()")}} 提供了一种通过[信标](/zh-CN/docs/Web/API/Beacon_API)向一个或多个指定的 URL 触发报告数据提交的方式，以便报告广告展示和点击情况。
```

File: files/zh-cn/web/api/fenced_frame_api/index.md [hunk 11]
```diff
@@ -0,0 +1,149 @@
...
+
+### 权限策略
+
+只有通过设置在 `<fencedframe>` 上的权限策略，才能启用专门设计为在其中使用的特定功能；在此上下文中，其他受策略控制的功能不可用。参见[围栏框架可用的权限策略](/zh-CN/docs/Web/HTML/Element/fencedframe#围栏框架可用的权限策略)查看更多信息。
```
Comment (yin1999) [inline files/zh-cn/web/api/fenced_frame_api/index.md:73]:
```suggestion
只有通过设置在 `<fencedframe>` 上的权限策略，才能启用专门设计为在其中使用的特定功能；在此上下文中，其他受策略控制的特性不可用。参见[围栏框架可用的权限策略](/zh-CN/docs/Web/HTML/Element/fencedframe#围栏框架可用的权限策略)查看更多信息。
```

File: files/zh-cn/web/api/fenced_frame_api/index.md [hunk 12]
```diff
@@ -0,0 +1,149 @@
...
+
+### HTTP 标头
+
+对于从 `<fencedframe>` 内部发出的任何请求，包括嵌入在 `<fencedframe>` 内的子 `<iframe>`，都会设置一个值为 `fencedframe` 的 {{httpheader("Sec-Fetch-Dest")}} 请求头。
```
Comment (yin1999) [inline files/zh-cn/web/api/fenced_frame_api/index.md:77]:
```suggestion
对于从 `<fencedframe>` 内部发出的任何请求，包括嵌入在 `<fencedframe>` 内的子 `<iframe>`，都会设置一个值为 `fencedframe` 的 {{httpheader("Sec-Fetch-Dest")}} 标头。
```

File: files/zh-cn/web/api/fenced_frame_api/index.md [hunk 13]
```diff
@@ -0,0 +1,149 @@
...
+
+```http
+Sec-Fetch-Dest: fencedframe
+```
+
+服务器必须为任何打算加载到 `<fencedframe>` 中或嵌入在 `<fencedframe>` 内的 `<iframe>` 中的文档设置一个值为 `fenced-frame` 的 {{httpheader("Supports-Loading-Mode")}} 响应头。
```
Comment (yin1999) [inline files/zh-cn/web/api/fenced_frame_api/index.md:83]:
```suggestion
服务器必须为任何打算加载到 `<fencedframe>` 中或嵌入在 `<fencedframe>` 内的 `<iframe>` 中的文档设置一个值为 `fenced-frame` 的 {{httpheader("Supports-Loading-Mode")}} 响应标头。
```

File: files/zh-cn/web/api/fenced_frame_api/index.md [hunk 14]
```diff
@@ -0,0 +1,149 @@
...
+
+```http
+Supports-Loading-Mode: fenced-frame
+```
+
+围栏框架对 HTTP 标头字段的其他影响如下：
+
+- [用户代理客户端提示](/zh-CN/docs/Web/HTTP/Client_hints#用户代理客户端提示)在封闭框架中不可用，因为它们依赖于[权限策略](/zh-CN/docs/Web/HTTP/Permissions_Policy)委托，这可能会被用来泄露数据。
```
Comment (yin1999) [inline files/zh-cn/web/api/fenced_frame_api/index.md:91]:
```suggestion
- [用户代理客户端提示](/zh-CN/docs/Web/HTTP/Client_hints#用户代理客户端提示)在围栏框架中不可用，因为它们依赖于[权限策略](/zh-CN/docs/Web/HTTP/Permissions_Policy)委托，这可能会被用来泄露数据。
```

File: files/zh-cn/web/api/fenced_frame_api/index.md [hunk 15]
```diff
@@ -0,0 +1,149 @@
...
+- 对从封闭框架内部打开的新浏览上下文强制执行严格的 [`Cross-Origin-Opener-Policy`](/zh-CN/docs/Web/HTTP/Headers/Cross-Origin-Opener-Policy) 设置，否则它们可能会被用来向其他源泄露信息。从封闭框架内部打开的任何新窗口都将设置 [`rel="noopener"`](/zh-CN/docs/Web/HTML/Attributes/rel/noopener) 和`Cross-Origin-Opener-Policy: same-origin`，以确保 {{domxref("Window.opener")}} 返回 `null` 并将其置于自己的浏览上下文组中。
```
Comment (yin1999) [inline files/zh-cn/web/api/fenced_frame_api/index.md:92]:
```suggestion
- 对从封闭框架内部打开的新浏览上下文强制执行严格的 [`Cross-Origin-Opener-Policy`](/zh-CN/docs/Web/HTTP/Headers/Cross-Origin-Opener-Policy) 设置，否则它们可能会被用来向其他源泄露信息。从围栏框架内部打开的任何新窗口都将设置 [`rel="noopener"`](/zh-CN/docs/Web/HTML/Attributes/rel/noopener) 和`Cross-Origin-Opener-Policy: same-origin`，以确保 {{domxref("Window.opener")}} 返回 `null` 并将其置于自己的浏览上下文组中。
```

File: files/zh-cn/web/api/fenced_frame_api/index.md [hunk 16]
```diff
@@ -0,0 +1,149 @@
...
+- 添加 [`Content-Security-Policy: fenced-frame-src`](/zh-CN/docs/Web/HTTP/Headers/Content-Security-Policy/fenced-frame-src) 来指定加载到 `<fencedframe>` 元素中的嵌套浏览上下文的有效来源。
+- 为了缓解隐私问题，[`Content-Security-Policy: sandbox`](/zh-CN/docs/Web/HTTP/Headers/Content-Security-Policy/sandbox) 自定义设置不能被围栏框架继承。要使封闭框架加载，需要指定无 `sandbox` CSP（这意味着以下值），或者指定以下沙箱值：
```
Comment (yin1999) [inline files/zh-cn/web/api/fenced_frame_api/index.md:94]:
```suggestion
- 为了缓解隐私问题，[`Content-Security-Policy: sandbox`](/zh-CN/docs/Web/HTTP/Headers/Content-Security-Policy/sandbox) 自定义设置不能被围栏框架继承。要加载围栏框架，需要指定无 `sandbox` CSP（这意味着以下值），或者指定以下沙箱值：
```

File: files/zh-cn/web/api/fenced_frame_api/index.md [hunk 17]
```diff
@@ -0,0 +1,149 @@
...
+  - `allow-same-origin`
+  - `allow-forms`
+  - `allow-scripts`
+  - `allow-popups`
+  - `allow-popups-to-escape-sandbox`
+  - `allow-top-navigation-by-user-activation`
+
+### `beforeunload` 和 `unload` 事件
+
+[`beforeunload`](/zh-CN/docs/Web/API/Window/beforeunload_event) 和 [`unload`](/zh-CN/docs/Web/API/Window/unload_event) 事件不能在围栏框架上触发，因为它们可以以页面删除时间戳的形式泄露信息。实施旨在消除尽可能多的潜在泄漏。
```
Comment (yin1999) [inline files/zh-cn/web/api/fenced_frame_api/index.md:104]:
```suggestion
[`beforeunload`](/zh-CN/docs/Web/API/Window/beforeunload_event) 和 [`unload`](/zh-CN/docs/Web/API/Window/unload_event) 事件不能在围栏框架上触发，因为它们可以以页面删除时间戳的形式泄露信息。实现旨在消除尽可能多的潜在泄漏。
```

File: files/zh-cn/web/api/fenced_frame_api/index.md [hunk 18]
```diff
@@ -0,0 +1,149 @@
...
+### 对其他接口的扩展
+
+- {{domxref("Navigator.deprecatedReplaceInURN()")}}
+  - : 在给定不透明 URN 或 `FencedFrameConfig` 的内部 `url` 属性所对应的映射 URL 中，替换指定的字符串。
+- {{domxref("Window.fence")}}
+  - : 返回当前文档上下文的一个 {{domxref("Fence")}} 对象实例。仅在嵌入在 `<fencedframe>` 内的文档中可用。
+
+## 注册和本地测试
+
+某些创建 {{domxref("FencedFrameConfig")}} 的 API 功能，如 {{domxref("Navigator.runAdAuction()")}}（[受保护的受众 API](https://developers.google.com/privacy-sandbox/private-advertising/protected-audience)）和 {{domxref("WindowSharedStorage.selectURL()")}}（[共享存储 API](/zh-CN/docs/Web/API/Shared_Storage_API)），以及其他功能如 {{domxref("Fence.reportEvent()")}}，要求你将你的网站注册到[隐私沙盒注册流程](/zh-CN/docs/Web/Privacy/Privacy_sandbox/Enrollment)中。如果你不注册，API 调用将失败，并在控制台中显示警告。
```
Comment (yin1999) [inline files/zh-cn/web/api/fenced_frame_api/index.md:124]:
```suggestion
某些创建 {{domxref("FencedFrameConfig")}} 的 API 特性，如 {{domxref("Navigator.runAdAuction()")}}（[受保护的受众 API](https://developers.google.com/privacy-sandbox/private-advertising/protected-audience)）和 {{domxref("WindowSharedStorage.selectURL()")}}（[共享存储 API](/zh-CN/docs/Web/API/Shared_Storage_API)），以及其他特性如 {{domxref("Fence.reportEvent()")}}，要求你将你的网站注册到[隐私沙盒注册流程](/zh-CN/docs/Web/Privacy/Privacy_sandbox/Enrollment)中。如果你不注册，API 调用将失败，并在控制台中显示警告。
```

File: files/zh-cn/web/api/fenced_frame_api/index.md [hunk 19]
```diff
@@ -0,0 +1,149 @@
...
+
+> [!NOTE]
+> 在 Chrome 中，你仍然可以在未注册情况下本地测试你的围栏框架代码。为了允许本地测试，请启用以下 Chrome 开发者标志：
```
Comment (yin1999) [inline files/zh-cn/web/api/fenced_frame_api/index.md:127]:
```suggestion
> 在 Chrome 中，你仍然可以在未注册情况下本地测试你的围栏框架代码。要允许本地测试，请启用以下 Chrome 开发者标志：
```

File: files/zh-cn/web/api/fenced_frame_api/index.md [hunk 20]
```diff
@@ -0,0 +1,149 @@
...
+{{Specifications}}
+
+## 浏览器兼容性
+
+{{Compat}}
+
+## 参见
+
+- [Fenced frames](https://developers.google.com/privacy-sandbox/private-advertising/fenced-frame)
+- [隐私沙盒](https://developers.google.com/privacy-sandbox)
```
Comment (yin1999) [inline files/zh-cn/web/api/fenced_frame_api/index.md:149]:
```suggestion
- developers.google.com 上的[围栏框架](https://developers.google.com/privacy-sandbox/private-advertising/fenced-frame)
- developers.google.com 上的[隐私沙盒](https://developers.google.com/privacy-sandbox)
```

**Review by yin1999 — CHANGES_REQUESTED** [review]
一些建议

**Review by yin1999 — APPROVED** [review]
Thank you @fuchunhui

**Pull request merged**